crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -242,7 +242,8 @@ impl<'a, T: Component> SystemParamFetch<'a> for ResState<T> {
             .get_populated_resource_column(state.component_id)
             .unwrap_or_else(|| {
                 panic!(
-                    "Requested resource does not exist: {}",
+                    "Resource requested by {} does not exist: {}",
```

Comment:
What do you think of "Resource {} does not exist, but was requested by {}"?

Reply:
I think as written it flows a bit more naturally.